app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FriendshipsController < ApplicationController
+  def index
+    @direct_friends = current_user.friendships.where(confirmed: true, user_id: current_user.id)
+    @inverse_friends = current_user.inverse_friendships.where(confirmed: true, friend_id: current_user.id)
+    @sent = current_user.friendships.where(user_id: current_user.id)
```

Comment:
[REQUIRED]
Instant variable `@sent` has been used in the views for displaying pending requests.
It's however displays even the confirmed request since the logic used here is not collect.
Let's update this query to search where confirmed is set to nil.
See this screenshot :point_down: 
![Screenshot from 2020-02-14 21-29-14](https://user-images.githubusercontent.com/44072711/74580348-28a4a900-4f71-11ea-9473-e958ad3c5bfc.png)
The same friendship request that has been confirmed is being displayed in the pending request section.